latch_cli/nextflow/workflow.py

```diff
@@ -225,7 +225,9 @@ def get_flag(name: str, val: Any) -> List[str]:
 
 def generate_nextflow_config(pkg_root: Path):
     config_path = Path(pkg_root) / "latch.config"
-    config_path.write_text(dedent("""\
+    config_path.write_text(
+        dedent(
+            """\
         process {
```

Comment:
can you indent this block so its on the same indentation as the opening block quote. same for all other instances where this got reformatted